pkgs/tools/misc/lazydocker/default.nix

```diff
@@ -21,6 +21,11 @@ buildGoModule rec {
 
   ldflags = [ "-s" "-w" "-X main.version=${version}" ];
 
+  passthru.tests.version = testers.testVersion {
+    package = lazydocker;
+    version = "Version: ${version}";
```

Comment:
I think we can just use the default here
```suggestion
```

Reply:
oh so this doesn't have to be super strict string matching? it just searches for the version inside the output?

Reply:
Yes, it searches full words that matches `version`